### PR TITLE
Fixed definition of fees collection

### PR DESCRIPTION
**Pull request by i-yasko**
## Summary
Update definition for fees collection

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/GatewayAccounts/SettlementSettings.yaml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 type: object
+nullable: true
```
Comment (ashkarpetin) [inline openapi/components/schemas/GatewayAccounts/SettlementSettings.yaml:2]: It is incorrect, gateway account settlement settings are not nullable:

https://user-api-docs.rebilly.com/tag/Gateway-accounts/operation/PutGatewayAccountFinancialSettings#!path=settlementSettings&t=request

File: openapi/components/schemas/FilterString.yaml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 type: string
+nullable: true
```
Comment (ashkarpetin) [inline openapi/components/schemas/FilterString.yaml:2]: It is not nullable for Shipping rates

File: openapi/components/schemas/GatewayAccounts/SettlementSettings.yaml (hunk 1 above)
Comment (i-yasko) [reply openapi/components/schemas/GatewayAccounts/SettlementSettings.yaml:2]: In this case settlementSettings definition should be separated. The fee can be created with  `settlementSettings: null`

File: openapi/components/schemas/Fees/BaseFee.yaml [hunk 3]
```diff
@@ -21,6 +21,8 @@ properties:
     minLength: 1
     maxLength: 255
     example: 'type:sale,capture;result:approved'
+    type: string
+    nullable: true
     allOf:
       - $ref: ../FilterString.yaml
```
Comment (ashkarpetin) [inline openapi/components/schemas/Fees/BaseFee.yaml:27]:
```suggestion
    nullable: true
    example: amount:1..100;currency:USD;bin:411111,444433
```

**Review by ashkarpetin — APPROVED** [review]

**Review by swilczynski — APPROVED** [review]

**Pull request merged**